src/axolotl/utils/models.py

```diff
@@ -115,9 +116,9 @@ def load_model(
         logging.info("patching with sdp attention")
         hijack_llama_sdp_attention()
 
-    if cfg.bf16:
+    if cfg.bf16 or cfg.bfloat16:
         torch_dtype = torch.bfloat16
-    elif cfg.load_in_8bit or cfg.fp16:
+    elif cfg.load_in_8bit or cfg.fp16 or cfg.float16:
```

Comment:
What is float16? Not sure if I asked this before and should've been some optimum_float16?

Reply:
related to fp16 and bf16 be HF and AMP related args,